### PR TITLE
Implement feature register and unitary test

### PR DESCRIPTION
**Pull request by LokiSchiffer**
Going to merge the sqlite helper to do the registry on the local database, and implement the unitary test

### CODE REVIEW

**Review by aapocode — APPROVED** [review]
All fine!

**Review by vivianachacon1 — APPROVED** [review]
Good job !!!!

File: app/src/androidTest/java/com/equipo4/mercadoapp/ExampleInstrumentedTest.java [hunk 1]
```diff
@@ -16,11 +22,37 @@
...
-    public void useAppContext() {
-        // Context of the app under test.
-        Context appContext = InstrumentationRegistry.getInstrumentation().getTargetContext();
-        assertEquals("com.equipo4.mercadoapp", appContext.getPackageName());
+    public void insertUser(){
+        User user = new User("Pepito Perez", "Pepito35", "pepito35@hotmail.com",
+                "prueba51", Long.parseLong("3101234567"));
+        long rows = helper.insert(User.TABLE, user.getContentValues());
+        //assertNotEquals(-1, rows);
+        assertEquals(rows, -1);
```
Comment (jpcorreap) [inline app/src/androidTest/java/com/equipo4/mercadoapp/ExampleInstrumentedTest.java:60]: Si el test se llama insertUser debería validar únicamente eso, recomendaría separar este assertEquals en un test aparte.

**Review by jpcorreap — CHANGES_REQUESTED** [review]

**Review by juansarmientog83 — APPROVED** [review]
very well!!

**Pull request merged**